From 6fb5ab5d2d7bebef70e49410b9f58a31c3db00da Mon Sep 17 00:00:00 2001
From: Ian Hinder <ian.hinder@aei.mpg.de>
Date: Sat, 22 Oct 2011 00:15:24 +0200
Subject: [PATCH] UserGuide: Change CVS references to SVN

Also remove a veritable CVS manual, leaving a very basic guide and a
pointer to the SVN documentation.
---
 doc/UsersGuide/Appendices.tex     |  352 +++++--------------------------------
 doc/UsersGuide/GettingStarted.tex |   41 ++---
 doc/UsersGuide/Notes.tex          |   15 +-
 3 files changed, 69 insertions(+), 339 deletions(-)

diff --git a/doc/UsersGuide/Appendices.tex b/doc/UsersGuide/Appendices.tex
index 518fb9b..ec3bba8 100644
--- a/doc/UsersGuide/Appendices.tex
+++ b/doc/UsersGuide/Appendices.tex
@@ -69,7 +69,7 @@ http://en.wikipedia.org/wiki/Cactus
   of a configuration (they don't change what binary will be produced).
 %  See Section~\ref{sec:configurations}.
 \item[checkout]
-  Get a copy of source code from CVS.%  See Section~\ref{sec:checkout}.
+  Get a copy of source code from SVN.%  See Section~\ref{sec:checkout}.
 \item[checkpoint]
   Save the entire state of a Cactus run to a file, so that the run can be
   restarted at a later time.
@@ -91,11 +91,11 @@ http://en.wikipedia.org/wiki/Cactus
   The \textit{Cactus Specification Tool}, which is the set of Perl
   scripts which parse the thorns' \texttt{.ccl} files, and generates the
   code that binds the thorn source files with the flesh.
-\item[CVS]
-  The \textit{Concurrent Versions System} is the favoured code
+\item[SVN]
+  \textit{Subversion} is the favoured code
   distribution system for Cactus.
   See Section %s~\ref{sec:checkout},
-  \ref{sec:Appendix.cvs}.
+  \ref{sec:Appendix.svn}.
 \item[domain decomposition]
   The technique of breaking up a large computational problem into parts
   that are easier to solve.  In Cactus, it refers especially to a decomposition
@@ -111,7 +111,7 @@ http://en.wikipedia.org/wiki/Cactus
 \item[flesh]
   The Cactus routines which hold the thorns together, allowing them to 
   communicate and scheduling things to happen with them. This is what you
-  get if you check out Cactus from our CVS repository.
+  get if you check out Cactus from our SVN repository.
 \item[friend] Interfaces that are \textit{friends}, share their collective
   set of protected grid variables.
   See Section~\ref{sec:Appendix.interface}. %%~\ref{subsec:interface_ccl}.
@@ -349,7 +349,7 @@ http://en.wikipedia.org/wiki/Cactus
   into a Cactus executable
   %(see Section~\ref{sec:gmtafobuanadco}, \ref{sec:cointh}). 
   Can also be used to determine which thorns
-  to check out from CVS.% (see Section~\ref{sec:checkout}).
+  to check out from SVN.% (see Section~\ref{sec:checkout}).
   A ThornList for each Cactus configuration lies in the configuration
   subdirectory of the Cactus {\tt configs} directory.
 \item[time bin]
@@ -1993,77 +1993,62 @@ comments will appear as ``anonymous''.
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-\chapter{Using CVS}
-\label{sec:Appendix.cvs}
-CVS is a version control system, which allows you to  keep
+\chapter{Using SVN}
+\label{sec:Appendix.svn}
+SVN is a version control system, which allows you to  keep
 old versions of files (usually source code), log of
 when, and why changes occurred, and who made them,  etc.
-Unlike the simpler systems, CVS does not just operate on one file at a
+SVN does not just operate on one file at a
 time or one directory at a time,  but
 operates  on  hierarchical collections of directories consisting of
-version controlled files.  CVS helps to  manage
+version controlled files.  SVN helps to  manage
 releases  and  to control the concurrent editing of source
-files among multiple  authors. CVS can be obtained from
-\url{http://www.cyclic.com}.
+files among multiple  authors. SVN can be obtained from
+\url{http://subversion.apache.org}, but is usually available on workstations, or
+can be easily installed using a package manager.
 
-A CVS \textit{repository} located on a \textit{server} may consist of an arbitrary
-number of \textit{modules}, which can be checked out (that is downloaded)
-independently. The Cactus flesh and the Cactus
-arrangements are organized as modules, their CVS \textit{server} is {\tt cvs.cactuscode.org}.
+An SVN \textit{repository} located on a \textit{server} contains a
+hierarchy of directory and files, and any subdirectory can be checked
+out independently.  The Cactus flesh and the Cactus arrangements are
+organized as repositories on the server {\tt svn.cactuscode.org}.  You
+can browse the contents of this repository using a web browser at the
+URL {\url http://svn.cactuscode.org}.
 
-\section{Essential CVS Commands}
+You do not need to know about SVN in order to download or update
+Cactus using the GetComponents script, though you must have SVN
+installed.  In order to contribute changes to Cactus files or your own
+thorns, which may also be stored in SVN, you will need a basic
+understanding of SVN.  For more information about 
+
+\section{Essential SVN Commands}
+
+Assuming that you have checked out Cactus using the GetComponents
+script, the following commands are the minimum you will need in order
+to work with SVN in Cactus.
 
 \begin{Lentry}
-\item[{\bf cvs login}]
-Logs into the repository. You will be prompted for a \textit{password}.
-This CVS command leaves a file {\tt .cvspass} in your
-home directory. There is no need to login every time you issue a CVS
-command, as long as this file exists. For a Cactus checkout, you have
-to log into the CVS server, using the CVS option {\tt -d} to specify {\tt CVSROOT}:\\
-{\tt cvs -d :pserver:cvs\_anon@cvs.cactuscode.org:/cactus login}
-
-\emph{Note}: that there is no ``logout'' command: if you log in with
-administrative rights from an arbitrary account, you should be aware
-that the password file enables subsequent administrative logins from
-that account. \emph{Delete the file if necessary}.
-
-\item[{\bf cvs checkout} \textit{modules} \ldots]
-This command  creates
-your private copy of the source for \textit{modules}. You can work
-with this copy  without  interfering  with  others'
-work.   At  least  one subdirectory level is always created: it does
-not write all files into your current directory but creates a
-directory. For Cactus, you need to either include the {\tt -d} options to
-specify the {\tt CVSROOT} directory and the CVS server, or specify them
-with an environment variable (see below). Once you
-have checked out the repository there is no need to include the {\tt
--d} option and its rather lengthy argument: the necessary information
-is contained in the local {\tt CVS/} directories.
-
-\item[{\bf cvs update}]
-Execute  this  command  from  \emph{within}  your  private
-source  directory  when  you  wish  to  update your
+
+\item[{\bf svn update}]
+Execute  this  command  from  \emph{within}  your  working
+directory  when  you  wish  to  update your
 copies of source  files  from  changes  that  other
 developers have made to the source in the repository.
 Merges are performed automatically when possible, a warning is issued
-if manual  resolution  is  required  for conflicting  changes. If your
-local copy is several versions behind the actual repository copy, CVS
-will \emph{refetch} the whole file instead of applying multiple
-patches.
+if manual  resolution  is  required  for conflicting  changes. 
 
-\item[{\bf cvs add} {\tt file}]
-Use this command to enroll new files in CVS records
+\item[{\bf svn add} {\tt file}]
+Use this command to enroll new files in SVN records
 of your working directory.  The files will be added
-to the  repository  the  next  time  you  run  `{\tt cvs
+to the  repository  the  next  time  you  run  `{\tt svn
 commit}'.
 
-\item[{\bf cvs commit} {\tt file}]
+\item[{\bf svn commit} {\tt file}]
 Use this command to add your local changes to the source to
 the repository and, thereby, making it publically available to
 checkouts and updates by other users. You cannot commit a
 newly created file unless you have \emph{added} it.
 
-\item[{\bf cvs diff} {\tt file}]
+\item[{\bf svn diff} {\tt file}]
 Show differences between a file in your working directory
 and  a file in the source repository, or between two revisions in
 source repository.  (Does not change either repository or working
@@ -2072,260 +2057,23 @@ directory.) For example, to see the difference between versions
 
 {\tt
 \begin{verbatim}
-cvs diff -r 1.8 1.9 foobar.c
+svn diff -r 1.8 1.9 foobar.c
 \end{verbatim}
 }
 
-\item[{\bf cvs remove} {\tt file}]
-Remove files from the source repository, pending  a {\tt cvs commit} on
+\item[{\bf svn remove} {\tt file}]
+Remove files from the source repository, pending  an {\tt svn commit} on
 the same files.
 
-\item[{\bf cvs status} {[}file{]}]
+\item[{\bf svn status} {[}file{]}]
 This command returns the current status of your local copy relative to
 the repository: e.g.\ it indicates local modifications and possible
 updates.
 
-\item[{\bf cvs import} {\tt repository tag1 tag2}]
-Import adds an entire source distribution (starting from the
-directory you issue the command in) to the repository directory.
-Use this command to add new arrangements to the Cactus 4.0 repository. The
-{\tt repository} argument is a directory name (or a path to a
-directory) and the CVS root directory for repositories; to obtain this
-directory on the CVS server, send a request to {\tt
-cactus@cactuscode.org}. {\tt tag1} and {\tt tag2} are two tags (vendor
- and release tags) that have to be supplied. For example, to add {\tt MyThorn}
-to the {\tt MyArrangement} arrangement, which may or may not already exist on
-the CVS repository
-
-{\tt
-\begin{verbatim}
-cvs -d :pserver:<name>@cvs.cactuscode.org:/cactus import MyArrangement/MyThorn
-start v1
-\end{verbatim}
-}
-
-After you import a thorn, you should check it out from the repository straight
-away, and only edit this version.
-
-\end{Lentry}
-
-\section{CVS Options}
-The CVS command line can include the following:
-\begin{Lentry}
-\item[{\bf cvs options}] which apply to the overall CVS program
-\item[{\bf a cvs command}]  which defines a particular task carried out by
-CVS
-\item[{\bf command options}] to specify certain working modes for the CVS
-command.
-\item[{\bf command arguments}] to specify which file to act on.
-\end{Lentry}
-
-The options must be put \emph{relative} to the CVS \emph{command} as the
-same option name can mean different things: CVS \emph{options} go to the
-\emph{left} of the CVS command, \emph{command options} go to the \emph{right}
-of the CVS command. Here is a list of essential CVS options:
-
-\begin{Lentry}
-
-\item[{\bf -d} \textit{cvs\_root\_directory}]
-Use \textit{cvs\_root\_directory} as the root directory  pathname  of
-the  master source repository.  Overrides
-the setting of the  {\tt CVSROOT}  environment  variable.
-This value should be specified as an absolute pathname.
-In the Cactus checkout procedure, you specify the Cactus CVS server:\\
-{\tt -d :pserver:cvs\_anon@cvs.cactuscode.org:/cactus/}
-
-\item[{\bf -z} \textit{compression-level}]
-When transferring files across the network use {\tt gzip}
-with compression level  \textit{compression-level}  to  compress  and
-decompress  data as it is transferred.
-Requires the presence of the GNU {\tt gzip} program  in
-the current search path at both ends of the link.
-
-\item[{\bf -n}]
-Do not change any file. Attempt to execute the CVS \textit{command} but
-only to issue reports. Does not remove, update, etc., any files. Very
-effective for testing.
-
-\item[{\bf -v}]
-Displays version information of the installed CVS.
-
-\item[{\bf -H} \textit{cvs-command}]
-Displays usage information about the specified CVS command. Without
-\textit{cvs-command}, a list of all available commands is returned.
-\end{Lentry}
-
-Here is a list of essential command options with the
-commands they are used with. They go after the CVS command. For a more
-complete list of all options, please refer to the manual page.
-
-\begin{Lentry}
-
-\item[{\bf -P}]
-Prune  (remove)  directories  that  are empty after being updated, on
-{\tt checkout}, or  {\tt update}.   Normally, an  empty  directory
-(one that is void of revision controlled files) is  left  alone.
-Specifying  {\tt -P} will cause these directories to be silently
-removed from the sources you have checked out.  This does not remove
-the directory from the repository, only from your checked out copy.
-
-\item[{\bf -m} \textit{"Text"}]
-Specify a logging message explaining changes, etc., on {\tt commit},
-{\tt import}. If you do not specify a message, your default editor
-is invoked to allow you to enter one.
-
-\item[\bf -d]
-Use this option with the {\tt update} command to create any
-directories if they are missing from your local copy. This is normally
-the case if another user has added files or directories to the
-repository. By default, the {\tt update} command only acts on files in
-your local copy. Note that omitting this option is a frequent cause of
-files missing during compilation.  (You can change this
-default behavior of CVS by putting a {\tt .cvsrc} in your home directory
-with the contents ``{\tt update -d}''.)
-
 \end{Lentry}
 
-\section{CVS Examples}
-We list some sample CVS commands to treat the most typical Cactus 4.0
-CVS situations.
-\begin{description}
-\item\textbf{Logging into the server}\newline
-{\tt cvs -d :pserver:cvs\_anon@cvs.cactuscode.org:/cactus
-login} \\ You will be asked for the password for user \textit{cvs\_anon}, which is {\tt anon}.
-
-\item\textbf{Checking out the code}\newline
-{\tt cvs -d :pserver:cvs\_anon@cvs.cactuscode.org:/cactus
-checkout Cactus}\\
-check out a CVS module named ``Cactus'', in this case it checks out the
-Cactus Computational Toolkit. A directory {\tt ./Cactus} is created if
-it doesn't already exist. If you perform a checkout on an already
-existing and locally modified copy of the module, CVS will try to merge the files
-with your local copy.
-
-\item\textbf{Updating a file or directory}\newline
-Assuming that you have a file {\tt ./foobar} in your checked out
-copy, you may perform a \\
-{\tt cvs status ./foobar}\\
-to inform yourself about the necessary updates, etc. To update the
-file issue \\
-{\tt cvs update ./foobar}\\
-If that was file was locally modified, CVS will try to merge the
-changes. Manual merging might be necessary and will be indicated by a
-CVS warning.
-
-\item\textbf{Updating a directory}\newline
-To recursively update the current directory and all subdirectories,
-type\\
-{\tt cvs update .}\\
-To update a directory {\tt ./mysources}, type\\
-{\tt cvs update ./path/to/mysources}
-
-\item\textbf{Committing a changed file}\newline
-To commit changes you have applied to your local copy, your file must be in
-sync with the repository: your changes must be done to the
-latest version, otherwise CVS will instruct you to perform an {\tt
-update} first. To commit changes made to a file {\tt ./foobar}, type\\
-{\tt cvs commit -m "Reason for the change" ./foobar}\\
-You may specify several files to commit.
-
-\item\textbf{Adding and committing a new file}\newline
-Adding a new file to the repository is a two fold procedure you first
-schedule the file for addition, then you commit it:\\
-{\tt cvs add ./newfoo}\\
-{\tt cvs commit -m "new few message" ./newfoo}
-
-\item\textbf{Creating a new thorn}\newline
-
-To add a new \textit{module} (e.g.\ an arrangement) to a Cactus repository, we
-first have to create a directory for you with the right permissions.
-Please contact {\tt cactus@cactuscode.org} providing the name of the
-requested module, and who should be able to commit changes to the module.
-
-To add the new module, change directory so that you are in the first directory
-that you want to commit to the repository. (For example,\ if you want to commit
-a new arrangement called {\tt MyArrange} then change directory to
-{\tt MyArrange}). Then type\\
-{\tt cvs -d :pserver:}\textit{your\_login}{\tt
-@cvs.cactuscode.org:<repository name> } import \textit{module\_name} {\tt start V1}\\
-(where {\tt start} and {\tt V1} are the vendor and release tags, which you could change to something different).
-
-To add a new \textit{directory} {\tt <new dir>} to an existing module (that you have write permissions for), either add the directory using\\
-{\tt cvs add <new dir>}\\
-and then recursing down adding all the new files and directories contained
-inside, or import the directory by changing directory to sit inside it, and then using\\
-{\tt cvs -d :pserver:}\textit{your\_login}{\tt
-@cvs.cactuscode.org:<repository\_name> } import {\tt <relative name> start V1}\\
-Where {\tt <relative name>} means the position of the directory within the module. (For example, if you have a module called \textit{AMod} which contains a
-directory \textit{BMod}, and you want to add \textit{CMod} inside \textit{BMod}, then change directory to \textit{BMod}, and use \textit{AMod/BMod} for the \textit{relative name}).
-
-
-\end{description}
-
-\section{Checking Out Flesh and Thorns with CVS}
-
-\begin{Lentry}
-\item[{\bf Login}] Prior to any CVS operation, you need to log into the Cactus
-  repository. For an anonymous checkout, type:\\
-  {\tt
-    cvs -d :pserver:cvs\_anon@cvs.cactuscode.org:/cactus login
-    }\\
-  You will be prompted for a password, which is {\tt anon}.
-\item[{\bf Checkout}] To obtain a fresh copy of Cactus, move to a directory
-  which does not contain a previously checked out version, and type
-  {\t
-    cvs -d :pserver:cvs\_anon@cvs.cactuscode.org:/cactus checkout Cactus
-    }
-  The CVS checkout procedure will create a directory called {\tt
-  Cactus} and install the code inside this directory.  From now, on we
-  will reference all directory names relative to {\tt Cactus}.
-
-\noindent
-  If you want to compile Cactus with thorns, you now need to checkout
-  separately the required arrangement (or \textit{arrangements})
-   into the {\tt arrangements} directory. To see the
-  available Cactus arrangements and thorns type
-  {\t
-    cvs -d :pserver:cvs\_anon@cvs.cactuscode.org:/cactus checkout -s
-  }
-  To check out an arrangement or thorn, go to the arrangements directory,  {\t cd arrangements},
-  and  for an arrangement type
-{\t
-  cvs checkout <arrangement\_name>
-  }
-        or for just one thorn
-{\t
-cvs checkout <arrangement\_name/thorn\_name>
-}
-
-To simplify this procedure, you may use {\t gmake checkout} in the Cactus
-home directory which provides menus to pick arrangements and thorns from.
-
-
-\item[{\bf Update}] To update an existing Cactus checkout (to patch in
-  possible changes, etc.), do the following \emph{within} the {\tt Cactus} directory.
-  {\t
-    cvs update
-    }
-  The update process will operate recursively downwards from your current position
-  within the Cactus tree. To update only on certain directories, change
-  into these directories and issue the update command.
-\item[{\bf CVS status}] To obtain a status report on the ``age'' of your
-  Cactus or arrangement routines (from your current directory position
-  downward), type
-  {\t
-    cvs status
-    }
-\item[{\bf non-anonymous CVS}] If you have an account at the central
-  repository ({\tt cvs.cactuscode.org}) and would like to perform
-  any of the operation above
-  \emph{non-anonymously}, replace {\tt cvs\_anon} by your login name
-  and provide the appropriate password during the CVS login
-  process. Depending on your permissions, you may then make commits to Cactus
-  or its arrangements.
-\item[{\bf Commits}] You need to perform a personalized login and have
-  proper permissions to commit code to the repository.
-\end{Lentry}
+For more information about using SVN, you can read the documentation
+provided at \url{http://svnbook.red-bean.com}.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
@@ -2415,12 +2163,6 @@ find the first subroutine in any file with this problem.}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-\chapter{ThornLists}
-
-\label{chap:th}
-
-This section still needs to be written.
-
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
diff --git a/doc/UsersGuide/GettingStarted.tex b/doc/UsersGuide/GettingStarted.tex
index 71b7a9f..caef407 100644
--- a/doc/UsersGuide/GettingStarted.tex
+++ b/doc/UsersGuide/GettingStarted.tex
@@ -23,16 +23,16 @@
 \section{Obtaining Cactus}
 \label{sec:checkout}
 
-Cactus is distributed, extended, and maintained using the free CVS
-software (\textit{Concurrent Versions System}: \url{http://www.cvshome.org}).
-CVS  allows many people to work on a large software project
+Cactus is distributed, extended, and maintained using the free Subversion
+software (\url{http://subversion.apache.org/docs/})
+SVN allows many people to work on a large software project
 together without getting into a tangle.
 Since Cactus thorns are distributed from several repositories on the
-main CVS site, and from a growing number of user sites, we provide a
-\texttt{GetCactus} script on our website for checking out the flesh 
+main SVN site, and from a growing number of user sites, we provide a
+\texttt{GetComponents} script on our website for checking out the flesh 
 and thorns. The script is available at
 
-\url{http://www.cactuscode.org/download/GetCactus}.
+\url{https://github.com/gridaphobe/CRL/raw/ET_2011_05/GetComponents}.
 
 The
 script takes as an argument the name of a file containing a \textit{ThornList},
@@ -41,33 +41,22 @@ that is a list of thorns with the syntax
 <\var{arrangement name}>/<\var{thorn name}>
 \end{alltt}
 
-If no filename is given, only the flesh is checked out.
-Optional directives in the ThornList indicate which CVS repository to fetch
-thorns from. The default is to take the thorns from the same repository as
-the flesh. A full description of ThornList syntax is provided in Appendix~\ref{chap:th}.
-ThornLists for example applications are provided on the Cactus website at
-
-%%% FIXME: url
+Optional directives in the ThornList indicate which repository to
+fetch thorns from. The ThornList is written in the {\em Component
+  Retrieval Language}, documented at
+\url{https://github.com/gridaphobe/CRL/wiki/Component-Retrieval-Language}.
 
 The same script can be used to checkout additional thorns, or to update 
 existing ones.
 
-The Cactus website also provides a form interface for direct download at:
-
-\url{http://www.cactuscode.org/cactus\_cgi-bin/download.pl}
-
-CVS experts who want to use raw CVS commands are directed to
-Appendix~\ref{sec:Appendix.cvs} for full instructions. For CVS novices,
-we also summarize in the Appendix basic CVS commands.
-
-The space required for an installation depends on the arrangements and
-thorns used. The flesh on its own requires less than 5 MB.
+The components that make up Cactus can also be checked out directly
+using SVN from \url{http://svn.cactuscode.org}.
 
 Another script, \texttt{MakeThornList}, can be used to produce a minimal
 ThornList from a given Cactus par file.  It needs a \emph{master} ThornList
 to be copied into your \texttt{Cactus} directory.
 
-See \url{http://www.cactuscode.org/toolkit/makeThornList/}.
+See \url{http://cactuscode.org/download/thorns/MakeThornList}.
 
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
@@ -80,7 +69,7 @@ following subdirectories:
 
 \begin{Lentry}
 
-\item[\texttt{CVS}] the CVS bookkeeping directory, present in every subdirectory
+\item[\texttt{.svn}] the SVN bookkeeping directory, present in every subdirectory
 
 \item[\texttt{doc}] Cactus documentation
 
@@ -91,7 +80,7 @@ following subdirectories:
 \item [\texttt{arrangements}] contains the Cactus arrangements. The arrangements
   (the actual ``physics'') are not supplied by just checking out just Cactus.
   If the arrangements you want to use are standard Cactus arrangements, or
-  reside on our CVS repository (\texttt{cvs.cactuscode.org}),
+  reside on our SVN repository (\texttt{svn.cactuscode.org}),
   they can be checked out in similar way to the flesh.
 \end{Lentry}
 
diff --git a/doc/UsersGuide/Notes.tex b/doc/UsersGuide/Notes.tex
index cf9afd4..3c031dc 100644
--- a/doc/UsersGuide/Notes.tex
+++ b/doc/UsersGuide/Notes.tex
@@ -44,11 +44,10 @@ in single processor mode. Please refer to the architecture section
 \item[CPP] C Preprocessor. For example, the GNU \texttt{cpp}.  These are
   normally provided on most platforms, and many C compilers have an option
   to just run as a preprocessor.
-\item[CVS] The \textit{Concurrent Versions System} is not needed
+\item[SVN] \textit{Subversion} is not needed
   to run/compile Cactus, but you are strongly encouraged to install
   this software to take advantage of the update procedures. It can be
-  downloaded from your favorite GNU site.  Tar files of each release are
-  also available.
+  downloaded from \url{http://subversion.apache.org}.  
 \end{Lentry}
 
 \noindent
@@ -903,8 +902,8 @@ The configuration options are stored in a file
 \item [\texttt{gmake <\var{config}>-configinfo}] displays the options
 of the configuration (\texttt{cat configs/<\var{config}>/config-info}).
 
-\item[\texttt{gmake <\var{config}>-cvsupdate}] updates the flesh and this
-configuration's thorns from the CVS repositories.
+%% \item[\texttt{gmake <\var{config}>-cvsupdate}] updates the flesh and this
+%% configuration's thorns from the CVS repositories.
 
 \item [\texttt{gmake <\var{config}>-delete}] deletes a configuration
 (\texttt{rm -r configs/<\var{config}>}).
@@ -1031,11 +1030,11 @@ arrangements and thorns.  For example, it can checkout all the thorns
 in any thornlist file found in the \texttt{thornlists} subdirectory of
 the Cactus root directory. % (usually \texttt{Cactus}).
 
-\item [\texttt{gmake cvsdiff}] shows differences between the checked out version of Cactus and that in the CVS repositories.
+%% \item [\texttt{gmake cvsdiff}] shows differences between the checked out version of Cactus and that in the CVS repositories.
 
-\item [\texttt{gmake cvsstatus}] shows status of the checked out version of Cactus, reporting which files have been modified or need updating.
+%% \item [\texttt{gmake cvsstatus}] shows status of the checked out version of Cactus, reporting which files have been modified or need updating.
 
-\item [\texttt{gmake cvsupdate}] updates flesh and all thorns from CVS repositories.
+%% \item [\texttt{gmake cvsupdate}] updates flesh and all thorns from CVS repositories.
 
 \item [\texttt{gmake configinfo}] prints configuration options for every
 configuration found in user's \texttt{configs} subdirectory.
-- 
1.7.5.4

